backend/windmill-api/openapi.yaml

```diff
@@ -8035,3 +8033,9 @@ components:
 
       required:
         - access_token
+
+    HubScriptKind:
```

Comment:
We could re-use it also in `schemas.Script` and `schemas.NewScript` but those two have `command`. Leaving it out for now